### PR TITLE
[ticket/14270] Purge cache when the installer is finished

### PR DESCRIPTION
**Pull request by CHItA**
[PHPBB3-14270](https://tracker.phpbb.com/browse/PHPBB3-14270)

### CODE REVIEW

**Comment by Nicofuma** [comment]
You forgot to update the service definition ;)

**Comment by CHItA** [comment]
Yeah, but it is wip still, i'm not sure if without a valid config.php `%cache.driver.class%` exists or not.

File: phpBB/install/startup.php [hunk 1]
```diff
@@ -81,4 +81,5 @@ function phpbb_include_updated($path, $phpbb_root_path, $optional = false)
 
 $phpbb_installer_container = $phpbb_installer_container_builder
 	->with_config_path($config_path)
+	->with_custom_parameters(array('cache.driver.class' => 'phpbb\cache\driver\file'))
```
Comment (CHItA) [inline phpBB/install/startup.php]: @Nicofuma: Do we need anything more advanced than this?

Comment (Nicofuma) [inline phpBB/install/startup.php]: I don't like this patch because it's a hack.... but it should work and I don't see any other "small" solution :/

Comment (marc1706) [inline phpBB/install/startup.php]: This is just the installer so there are things that are really more important than manually specifying the cache type.

Comment (Nicofuma) [inline phpBB/install/startup.php]: The real issue is that the cache and php file storage are separated but we rely on the cache to purge the stored php files. We need to build a proper API for that but that will not happen before 3.3

**Pull request merged**